src/main/java/ome/io/bioformats/OmeroPixelsPyramidReader.java

```diff
@@ -86,6 +86,7 @@ public void setId(String id) throws FormatException, IOException
     {
         log.debug("setId(" + id + ")");
         super.setId(id);
+        canSeparateSeries = false;
```

Comment:
Should the call to `canSeparateSeries` happen earlier in the stack? From https://github.com/openmicroscopy/bioformats/pull/3202, it looks like other readers override the flag in the constructor.

Reply:
Yeah, it should have been before the setId.